packages/field-base/src/field-mixin.js

```diff
@@ -79,14 +79,15 @@ export const FieldMixin = (superclass) =>
       this._helperController = new HelperController(this);
       this._errorController = new ErrorController(this);
 
-      this._labelController.addEventListener('label-changed', (event) => {
-        const { hasLabel, node } = event.detail;
-        this.__labelChanged(hasLabel, node);
+      this._labelController.addEventListener('node-changed', (event) => {
+        const { hasNode, node } = event.detail;
+        this.__labelChanged(hasNode, node);
       });
 
-      this._helperController.addEventListener('helper-changed', (event) => {
-        const { hasHelper, node } = event.detail;
-        this.__helperChanged(hasHelper, node);
+      this._helperController.addEventListener('node-changed', (event) => {
+        const { hasNode, node } = event.detail;
+        this.toggleAttribute('has-helper', hasNode);
```

Comment:
This listener is needed because the controller no longer takes care of setting host attribute.
See also related `LabelMixin` change where `has-label` attribute is set in the same way.